### ISSUE TITLE
[Doc] inconsistent code style of css hex color

### ISSUE THREAD

**Issue by kmsheng** [angular/angular#23113]
https://angular.io/tutorial/toh-pt5#heroescomponent

<img width="312" alt="screen shot 2018-04-01 at 12 07 14 pm" src="https://user-images.githubusercontent.com/880569/38169760-91e0e4de-35a5-11e8-9fbb-d3f349637c1d.png">

**Comment by alexeagle** [comment]
What's inconsistent about it? The capitalization of the A-F?

**Comment by kmsheng** [comment]
@alexeagle That's right. Either choose uppercase or lowercase for keeping the consistency would be better. This might seem to be trivial, but it's still something that can be improved.

**Comment by jenniferfell** [comment]
I couldn't find any recommendation in the Google Dev Doc Style Guide. Android dev doc uses lowercase. That's easier to type, so it's a good default. Any strong preference @kmsheng @jelbourn ?

**Comment by kmsheng** [comment]
@jenniferfell I was personally following this one
https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation

**Comment by jelbourn** [comment]
The internal Google style is to use all lowercase for what it's worth
